diff --git a/webserver/core/lib/iec_std_lib.h b/webserver/core/lib/iec_std_lib.h
index 56fe14b..68441bc 100755
--- a/webserver/core/lib/iec_std_lib.h
+++ b/webserver/core/lib/iec_std_lib.h
@@ -236,7 +236,7 @@ static inline IEC_TIMESPEC __time_to_timespec(int sign, double mseconds, double
 
 
 /* NOTE: The following function was turned into a macro, so it could be used to initialize the initial value of TOD (TIME_OF_DAY) variables */
-/* NOTE: many (but not all) of the same comments made regarding __time_to_timespec() are also valid here, so go and read those comments too!
+/* NOTE: many (but not all) of the same comments made regarding __time_to_timespec() are also valid here, so go and read those comments too! */
 /*
 static inline IEC_TIMESPEC __tod_to_timespec(double seconds, double minutes, double hours) {
   IEC_TIMESPEC ts;
@@ -581,7 +581,7 @@ static inline STRING __time_to_string(TIME IN){
     div_t days;
     /*t#5d14h12m18s3.5ms*/
     res = __INIT_STRING;
-    days = div((int &)IN.tv_sec, SECONDS_PER_DAY);
+    days = div((int)IN.tv_sec, SECONDS_PER_DAY);
     if(!days.rem && IN.tv_nsec == 0){
         res.len = snprintf((char*)&res.body, STR_MAX_LEN, "T#%dd", days.quot);
     }else{
